electron/generators/__snapshots__/diograph-generator.snapshot.test.js.snap

```diff
@@ -0,0 +1,70 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`diograph-generator generateDiograph generates diograph from example folder 1`] = `
+Object {
+  "uuid-0": Object {
+    "created": "2020-04-02T17:57:14.568Z",
+    "date": "2008-11-01T21:15:11.000Z",
+    "id": "uuid-0",
+    "image": "/Users/op/2019/diory-browser-electron/electron/readers/example-folder/other-image.jpg",
+    "latitude": 43.464455,
+    "longitude": 11.881478333333334,
+    "modified": "2020-04-02T17:57:14.569Z",
+  },
+  "uuid-1": Object {
+    "created": "2020-04-02T17:57:14.572Z",
+    "date": "2019-11-25T19:58:24.000Z",
+    "id": "uuid-1",
+    "image": "/Users/op/2019/diory-browser-electron/electron/readers/example-folder/some-image.jpg",
```

Comment:
This is why we need docker environment 😄  As you can imagine, this test won't pass on my computer...

But this is a good starting point, we can fix this later...